describealign.py

```diff
@@ -1126,10 +1126,10 @@ def main_gui():
   config_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'config.ini')
   sg.theme('Light Blue 2')
 
-  all_audio_file_types = [('All Audio File Types', '*.' + ';*.'.join(AUDIO_EXTENSIONS)),]
-  all_video_file_types = [('All Video File Types', '*.' + ';*.'.join(VIDEO_EXTENSIONS)),]
+  all_audio_file_types = [('All Audio File Types', '*.' + ' *.'.join(AUDIO_EXTENSIONS)),]

```

Comment:
This breaks it on Windows, but that can be fixed with a platform check (like on line 74).

Reply:
Is this not handled by `file_fix` on 1139?

Reply:
That just deals with a bug in this bit of PySimpleGUIWx's code:
`def convert_tkinter_filetypes_to_wx(filetypes):
    wx_filetypes = ''
    for item in filetypes:
        filetype = item[0] + ' (' + item[1] + ')|'+ item[1]
        wx_filetypes += filetype
    return wx_filetypes
`
The bug is it doesn't add the separator '|' between filetypes when it appends them to wx_filetypes, so file_fix just prepends it to every filetype after the first.
We can just do another if IS_RUNNING_WINDOWS, else here though and it should be fine.  Also I'm a bit surprised you were seeing a problem with the semicolon implementation on Mac/Linux, as I thought I'd tested it on there with the old version of PySimpleGuiQt.  Does it only need to be spaces for the new version or something?

Reply:
It worked fine on mac but on linux (ubuntu, opensuse, and a few other distros), it has to be space. space worked for mac as well, and [the QT docs](https://doc.qt.io/qt-6/qfiledialog.html#setNameFilters) only use spaces. I'll test windows and see about implementing that fix

Reply:
Fixed in 5410fb5